Lib/test/test_argparse.py

```diff
@@ -4958,6 +4958,51 @@ def custom_type(string):
     version = ''
 
 
+class TestHelpUsageLongSubparserCommand(TestCase):
+    """Test that subparser commands are formatted correctly in help"""
+
+    def test_parent_help(self):
+        def custom_formatter(prog):
+            return argparse.RawTextHelpFormatter(prog, max_help_position=50)
+
+        parent_parser = argparse.ArgumentParser(
+                prog='PROG',
+                add_help=False,
+                formatter_class=custom_formatter
+        )
+        main_parser = argparse.ArgumentParser(
+            formatter_class=custom_formatter)
+
+        cmd_subparsers = main_parser.add_subparsers(title="commands",
+                                                    metavar='CMD',
+                                                    help='command to use')
+        cmd_parser = cmd_subparsers.add_parser("add",
+                                               help="add something",
+                                               parents=[parent_parser])
+        cmd_parser.add_subparsers(title="action", dest="action_command")
```

Comment:
Is this line needed?

I now wonder what is the role of `parent_parser` and whether this test can be simplified  further. Try to write a minimal code that would fail without the fix and pass with the fix.